src/components/LoansByCategory/MFIRecommendations/MFIHero.vue

```diff
@@ -2,104 +2,118 @@
 	<div class="tw-w-full tw-mb-2">
 		<div class="tw-flex tw-items-center tw-gap-1 tw-px-1 lg:tw-px-2">
 			<!-- eslint-disable max-len -->
-			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="mfi_recommendations">
-			<h3 class="tw-text-h3 tw-text-primary tw-whitespace-nowrap">
-				Lending partner spotlight
+			<img src="@/assets/images/mfi_recommendations/megaphone.svg" alt="mfi_recommendations">
+			<h3 class="tw-text-h4 tw-text-primary tw-whitespace-nowrap">
+				IMPACT AT KIVA
 			</h3>
 		</div>
 		<div class="tw-mt-2 lg:tw-mt-4 tw-px-2">
-			<h2 class="tw-text-h2 tw-text-primary">
-				Fundación Pro Mujer
-			</h2>
-			<h6 class="tw-text-subhead tw-text-primary">
-				15 years with Kiva, 52,908 borrowers supported
+			<h2 v-html="headline" class="tw-text-h2 tw-text-primary"></h2>
+			<h6 class="tw-text-small tw-text-primary tw-mt-1">
+				{{ subheadline }}
 			</h6>
 		</div>
-		<div class="tw-flex tw-flex-col md:tw-flex-row tw-my-3">
+		<div class="tw-flex tw-flex-col md:tw-flex-row tw-my-3 tw-items-center">
 			<div class="md:tw-w-2/3 md:tw-px-2">
 				<div class="tw-relative">
 					<!-- eslint-disable max-len -->
-					<img class="md:tw-rounded tw-w-full" src="@/assets/images/mfi_recommendations/fundacionpromujer.jpg" alt="fundacionpromujer">
-					<span
-						class="tw-absolute tw-bottom-1 tw-left-3 tw-rounded tw-bg-tertiary
-							tw-px-2 tw-py-1 tw-flex tw-items-center tw-text-h4"
-					>
-						<kv-material-icon
-							class="tw-h-2.5 tw-w-2.5 tw-mr-0.5"
-							:icon="mdiMapMarker"
-						/>
-						BOLIVIA
-					</span>
+					<div class="tw-rounded tw-p-2 tw-bg-white tw-border" style="border-color: #C4C4C4;">
+						<img class="tw-rounded tw-w-full" :src="imageRequire(`./${partnerImage}`)">
+
+						<div class="tw-flex tw-items-center tw-mt-3 tw-mb-2 tw-gap-2">
+							<img src="@/assets/images/mfi_recommendations/leaf.svg" alt="mfi_recommendations">
```

Comment:
`alt` should be readable, maybe "MFI Recommendation"?

Reply:
Added

Reply:
Not seeing the change on this line, still coming?